.github/ISSUE_TEMPLATE/backlog_task.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Backlog Task
+about: Non-story related task, typically foundational.  Can be used for both research and engineering tasks.
+title: ''
+labels: foundational
+assignees: ''
+
+---
+
+# Backlog Task
+
+Add a title and description here
+
+## Completion Criteria
+
+- [ ] What is the criteria for calling this task complete?
+
+## Tasks
+
+- [ ] What tasks need to be done to fulfill the completion criteria?
+
+### Other notes
```

Comment:
1. Title case for consistency? (e.g. "Other Notes")
2. H2 instead of H3? (Presuming it's not a section within Tasks)

Reply:
@JohnNKing These have been updated